WooCommerce/Classes/Authentication/AuthenticationManager.swift

```diff
@@ -289,6 +289,22 @@ extension AuthenticationManager: WordPressAuthenticatorDelegate {
             return navigationController.show(mismatchAccountUI, sender: nil)
         }
 
+        if let matchedSite = matcher.matchedSite(originalURL: siteURL),
+           matchedSite.isWooCommerceActive == false {
+            let viewModel = NoWooErrorViewModel(
```

Comment:
Super Nit: 
What do you think about extracting the view model creation code to a private function?

IMO, this will make it easier to understand all the info/values `NoWooErrorViewModel` needs/uses.

Reply:
I'm not sure how to extract this without having to send a lot of parameters to the method, which will then create the same effect 😅 I'll keep this for now if you don't mind.

Reply:
Makes sense. 😅👍